subtitles.go

```diff
@@ -327,7 +340,9 @@ func (l Line) String() string {
 	for _, i := range l.Items {
 		texts = append(texts, i.Text)
 	}
-	return strings.Join(texts, " ")
+	t := strings.Join(texts, " ")
```

Comment:
This feels hacky, I think if we implement the rule `trim spaces on input, add in-between spaces on output`, we don't need this logic anymore

Reply:
yep, did felt hacky. Removed it